<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TuloveroDialog</class>
 <widget class="QDialog" name="TuloveroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tuloveron kirjaaminen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tutustu huolella verotukseen liittyviin ohjeisiin ja täydennä alla oleva laskelma.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Vain yksinkertaisimmissa tapauksissa ohjelman tekemä laskelma on sellaisenaan oikea.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="1">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="KpEuroEdit" name="yleveroEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Verotettava tulos</string>
       </property>
       <property name="buddy">
        <cstring>tulosEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="KpEuroEdit" name="jaaveroaEdit"/>
     </item>
     <item row="8" column="1">
      <widget class="KpEuroEdit" name="ennenYlea"/>
     </item>
     <item row="15" column="1">
      <widget class="KpEuroEdit" name="veroEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="KpEuroEdit" name="ennakkoYle"/>
     </item>
     <item row="6" column="1">
      <widget class="KpEuroEdit" name="tappioEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Vähennyskelpoiset kulut</string>
       </property>
       <property name="buddy">
        <cstring>vahennysEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="KpEuroEdit" name="maksettavaYle"/>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Maksetut ennakkoverot</string>
       </property>
       <property name="buddy">
        <cstring>maksetutEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Lopullinen verotettava tulos</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="KpEuroEdit" name="maksetutEdit"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Yle-vero</string>
       </property>
       <property name="buddy">
        <cstring>yleveroEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Verotettava tulos ennen Yle-veroa</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="KpEuroEdit" name="tulosEdit"/>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Maksamaton tulovero</string>
       </property>
       <property name="buddy">
        <cstring>jaaveroaEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tuloveron määrä</string>
       </property>
       <property name="buddy">
        <cstring>veroEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="KpEuroEdit" name="ylenJalkeen"/>
     </item>
     <item row="16" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Vähennettävä aiempi tappio</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Veronalainen tulo yhteensä</string>
       </property>
       <property name="buddy">
        <cstring>tuloEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KpEuroEdit" name="vahennysEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="KpEuroEdit" name="tuloEdit"/>
     </item>
     <item row="13" column="1">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Ennakkoon maksettu Yle-vero</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Maksamaton Yle-vero</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KpEuroEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpeuroedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tuloEdit</tabstop>
  <tabstop>vahennysEdit</tabstop>
  <tabstop>tulosEdit</tabstop>
  <tabstop>tappioEdit</tabstop>
  <tabstop>ennenYlea</tabstop>
  <tabstop>yleveroEdit</tabstop>
  <tabstop>ennakkoYle</tabstop>
  <tabstop>maksettavaYle</tabstop>
  <tabstop>ylenJalkeen</tabstop>
  <tabstop>veroEdit</tabstop>
  <tabstop>maksetutEdit</tabstop>
  <tabstop>jaaveroaEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TuloveroDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TuloveroDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
